src/core/connection.c

```diff
@@ -43,18 +43,37 @@ typedef struct QUIC_RECEIVE_PROCESSING_STATE {
 } QUIC_RECEIVE_PROCESSING_STATE;
 
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
+BOOLEAN
+QuicConnApplyNewInternalSettings(
+    _In_ QUIC_CONNECTION* Connection,
+    _In_ BOOLEAN OverWrite,
+    _In_ BOOLEAN CopyExternalToInternal,
+    _In_ const QUIC_SETTINGS_INTERNAL* NewSettings
+    );
+
 _IRQL_requires_max_(PASSIVE_LEVEL)
 BOOLEAN
 QuicConnApplyNewSettings(
     _In_ QUIC_CONNECTION* Connection,
     _In_ BOOLEAN OverWrite,
     _In_ BOOLEAN CopyExternalToInternal,
-    _In_range_(FIELD_OFFSET(QUIC_SETTINGS, DesiredVersionsList), UINT32_MAX)
```

Comment:
Why remove this?

Reply:
Anything using QUIC_SETTTINGS_INTERNAL does not need a size.